src/test/java/cucumber/pageObjects/ProductListPage.java

```diff
@@ -0,0 +1,18 @@
+package cucumber.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ProductListPage {
+    WebDriver driver;
+
+    public ProductListPage(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public void selectProduct(String productName) {
+        WebElement product = driver.findElement(By.xpath("//a[@title=\"" + productName + "\"]"));
```

Comment:
String.format looks nicer, especially if you will have some more parameters or if you will plan to reuse this xpath